src/requireConfig.ts

```diff
@@ -0,0 +1,11 @@
+import { FirebaseError } from "./error";
+import { Options } from "./options";
+
+export async function requireConfig(options: Options): Promise<void> {
+  await Promise.resolve(); // Allows this function to remain `async`.
```

Comment:
Lol, now I'm wondering why this was ever async in the first place.

Reply:
lol amazing hack